misc/tem_helpers.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+
+from definitions import ROOT_DIR
+
+sys.path.append(os.path.join(ROOT_DIR, 'tem', 'script'))
+
+import numpy as np
+import numpy.typing as npt
+
+from tem.script.model import TopicEvolution
```

Comment:
If we already append the path above, we can also write these imports without tem.script, however, this is optional, as this extended form also helps the reader to follow

Reply:
good point, i didn't realize that. but your argument of having it be more readable this way is even more convincing to I'll leave it as it is🙈